pkgs/top-level/python-packages.nix

```diff
@@ -9367,6 +9367,29 @@ in {
     };
   };
 
+  subliminal = buildPythonApplication rec {
```

Comment:
We do not put python applications here, as they only needs to be built against a particular version of python.
Please move this expression out of this file and reference it directly in `all-packages.nix`